extensions/amp-story/1.0/page-advancement.js

```diff
@@ -863,15 +918,14 @@ class MediaBasedAdvancement extends AdvancementConfig {
    */
   static fromAutoAdvanceString(autoAdvanceStr, win, rootEl) {
     try {
-      const element = rootEl.querySelector(`#${
-        escapeCssSelectorIdent(autoAdvanceStr)
-      }`);
-
-      if (!element) {
+      const elements = rootEl.querySelectorAll(
```

Comment:
nit: factor out the escaped ID?

Reply:
The linter doesn't understand that unfortunately : (

Reply:
wow, really?!  that's.... annoying 🙁